src/modules/ekf2/EKF/optflow_fusion.cpp

```diff
@@ -148,8 +148,8 @@ void Ekf::fuseOptFlow()
 			}
 		}
 
-		SparseVector24f<0,1,2,3,4,5,6> Hfusion(H);
-		Vector24f Kfusion = P * Hfusion / _aid_src_optical_flow.innovation_variance[index];
+		SparseVectorState<0,1,2,3,4,5,6> Hfusion(H);
```

Comment:
@dagar The sparse vector optimization is still in the way if we want to be able to swap state blocks